ui/packages/consul-ui/mock-api/v1/acl/binding-rules

```diff
@@ -0,0 +1,30 @@
+[
+  ${
+    range(
+      env(
+        'CONSUL_BINDING_RULE_COUNT',
+        Math.floor(
+          (
+            Math.random() * env('CONSUL_BINDING_RULE_MAX', 10)
+          ) + parseInt(env('CONSUL_BINDING_RULE_MIN', 1))
+        )
+      )
+    ).map(
+      function(item, i) {
+        return `
+          {
+            "ID": "${fake.random.uuid()}",
+            "Description": "${fake.lorem.sentence()}",
+            "AuthMethod": "${fake.hacker.noun()}-${i}",
+            "Selector": "serviceaccount.namespace==default and serviceaccount.name!=vault",
+            "BindType": "${fake.helpers.randomize(['service', 'node', 'role'])}",
```

Comment:
Could we randomize the `default` and `vault` just to give it a bit of 'variety' when you view the listing

Reply:
Updated to be a faker noun.